packages/moonstone/ExpandableItem/ExpandableItem.js

```diff
@@ -332,9 +332,9 @@ const ExpandableItemBase = kind({
 			// When disabled, don't attach an event
 			if (!disabled) {
 				if (open) {
-					onClose(ev);
+					onClose && onClose(ev);
```

Comment:
Although this is valid code, it does violate our linting rules and needs to be updated.